connaisseur/trust_data.py

```diff
@@ -103,19 +106,11 @@ def validate_signature(self, keystore: KeyStore):
             pub_key = keystore.get_key(key_id)
             sig = signature["sig"]
 
-            if not pub_key:
-                raise NotFoundException(
-                    "couldn't find right public for trust data.",
```

Comment:
Why has this been removed?

Reply:
should the key with `key_id` not exist, the `KeyStore.get_key` function will already throw an exception. should the content behind the `key_id` be empty, then an error while loading the key (`KeyStore.update`) would have caught that. Same is true for the `pub_root_key` during initialization of the `KeyStore`. In the end, this is dead code and was removed.